### PR TITLE
Made the documentation more user-friendly #1029

### PR DESCRIPTION
**Pull request by ankit7241**
1. Remove warp 1, warp 2 comparison
 2. Add quickstart in place of the comparison
 4. The page https://nethermindeth.github.io/warp/ is redundant. The first page in the documentation should be quickstart.

Changed the Welcome page from this :
image

to this :
image
image
image
image
image
image
image
image

Took reference from : https://github.com/NethermindEth/warp#quickstart 

3. Move installation and contributing from markdown files to the documentation

Changed data :

image
image
image
image

Rest all is same as before in this page

Took reference from :
https://github.com/NethermindEth/warp/blob/develop/installation.md and
https://github.com/NethermindEth/warp/blob/develop/dependencies.md

And later on deleted these file from github as instructed here :
image

Do let me know if there are some changes required.

### CODE REVIEW

**Comment by ankit7241** [comment]
Sorry, forgot to add the images of "Warp Installation Method 2 (from source/for devs)" . It is also still present there.

image
image
image

**Comment by rjnrohit** [comment]
It's great that you proceed forward with the website update, also pls look at this: 
https://github.com/NethermindEth/warp/pull/1008#issuecomment-1500263574

File: docs/docs/intro.md [hunk 1]
```diff
@@ -1,23 +1,173 @@
...
 
-# Welcome to Warp 2.0
+<!-- # Welcome to Warp 2.0
 
 We are excited to announce the second version of Warp, now designed to transpile your Solidity code directly into Cairo.
 
 Warp 1 set out to show that compiling Solidity code into Cairo was possible, and paved the way for developers to access the benefits of Starknet without needing to master Cairo. Using everything we learned from Warp 1, we have written a new version adding vast improvements to contract efficiency and user experience. In this blog, we will talk through the improvements made to Warp, transpile OpenZeppelin’s ERC20 contract, and describe future plans for the project.
 
-### Warp 2 vs Warp 1
+#### Want to know more on Warp 2.0? Read the blog [here](https://medium.com/nethermind-eth/warp-2-0-transpiling-directly-from-solidity-to-cairo-9bf41a6d26ee). -->
```
Comment (piwonskp) [inline docs/docs/intro.md]: Let's add a short introduction before Quickstart:
```
# Welcome to Warp!

Warp is a transpiler from Solidity to Cairo. It allows you to deploy your Solidity contracts on Starknet.
```

Comment (ankit7241) [reply docs/docs/intro.md]: Hey @piwonskp , I made the required changes. Please review it and let me know if any more changes are required.

image

**Comment by piwonskp** [comment]
Good work! Let's add a few more changes:
* Please move contributing.md and CLI commands to documentation as described in https://github.com/NethermindEth/warp/pull/1008#issuecomment-1500263574. 
* Remove docker, CLI commands sections from readme
* Make a list of quick links: Documentation, Installation and Contributing
* Add an alternative method of installation using docker to contributing page. It should include the usual apple silicon warning. Steps necessary to start development with docker:

1. Clone repository and `cd warp/`
2. Build docker containers:
```
docker-compose up
```
3. Install javascript dependencies :
```
docker-compose exec warp yarn
```
4. Compile the project:
```
docker-compose exec warp yarn build
```
5. Build warplib:
```
docker-compose exec warp yarn warplib
```
6. Test the installation worked by transpiling an example ERC20 contract:
```
bin/warp transpile exampleContracts/ERC20.sol
```

File: docs/docs/getting_started/a-usage-and-installation.mdx [hunk 2]
```diff
@@ -6,49 +6,93 @@ title: Installation and Usage
...
+```bash
+sudo apt install python3.9 python3.9-venv python3.9-dev
+```
+
+Or you can just install python3.9 and then install python-dev-tools using pip.
+
 2. Install Python3.9 with dev dependencies (`python3.9 python3.9-venv python3.9-dev`) into your base env.
    If you do not have the dev dependencies installed the installation will fail.
 
 ### Warp Installation Method 1
```
Comment (piwonskp) [inline docs/docs/getting_started/a-usage-and-installation.mdx]: Let's remove `installation method 2` and rename it to `Installation` after adding `Installation` to contributing page

File: docs/docs/intro.md [hunk 3]
```diff
@@ -1,23 +1,169 @@
 ---
-title: Welcome
+title: QuickStart
```
Comment (piwonskp) [inline docs/docs/intro.md]: Let's keep the page title
```suggestion
title: Welcome
```

File: docs/docs/intro.md [hunk 4]
```diff
@@ -1,23 +1,169 @@
...
+docker-compose up
+```
+
+### Transpile
+
+```bash
+docker-compose exec warp warp transpile exampleContracts/ERC20.sol
+```
+
+It's best to copy the contract/repo to the warp directory so it is available in container via volume. Use contract's paths relative to warp root. For example, assuming you've copied your project to `warp/projects/myproject` you can replace `exampleContracts/ERC20.sol` with `projects/myproject/mycontract.sol` in the above command.
```
Comment (piwonskp) [inline docs/docs/intro.md]: Please remove everything below the line. Those sections should be moved to the respective places in the documentation

**Comment by piwonskp** [comment]
Let's replace Quickstart in readme and docs with a new, easier method of running warp:
```
docker run --rm -v "$PWD:/dapp" nethermind/warp transpile ERC20.sol
```

**Comment by piwonskp** [comment]
> Let's replace Quickstart in readme and docs with a new, easier method of running warp:
> 
> ```
> docker run --rm -v "$PWD:/dapp" nethermind/warp transpile ERC20.sol
> ```

I've added those instructions to the readme on develop so please merge develop to your branch and update documentation accordingly

**Review by piwonskp — APPROVED** [review]

**Pull request merged**